README.md

```diff
@@ -28,6 +28,8 @@ Example patterns:
 | `$ELEMENT.innerHTML` | ``el.innerHTML = "<img src='x' onerror='alert(`XSS`)'>";`` |
 | `$TOKEN.SignedString([]byte("..."))` | `ss, err := token.SignedString([]byte("HARDCODED KEY"))` |
 
+→ [see more example patterns in the semgrep-rules repository](https://github.com/returntocorp/semgrep-rules)
```

Comment:
I think this works fine but how about sending people to https://semgrep.live/r/ ? Just seems easier to browse than the GitHub file navigation IMO.

Reply:
@DrewDennison would have to comment--AFAIK that's not ready to be advertised yet?

Reply:
@pabloest I think https://semgrep.live/r/ is still too functional without enough context and explanation for direct linking. I would like to hustle this weekend on making it so it's ready and I know Luke has been asking for a new design on that page. If you want to spec it out with me, that would be great

Reply:
Happy to help, @DrewDennison. We can chat off-thread.